tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -0,0 +1,119 @@
+@api @skipOnOcV10
+Feature: add users to group
+  As a admin
+  I want to be able to add users to a group
+  So that I can give a user access to the resources of the group
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario: adding a user to a group
```

Comment:
a then step to check if the added users are in the group members list would make the scenario better imo